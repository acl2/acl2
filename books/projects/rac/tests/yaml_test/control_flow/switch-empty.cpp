// RAC begin

int foo(int x) {

  switch (x) {
  }

  return x;
}
