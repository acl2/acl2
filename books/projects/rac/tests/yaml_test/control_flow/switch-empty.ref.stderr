switch-empty.cpp:6 (2-3): syntax error, unexpected '}', expecting CASE or DEFAULT
 6 |   }
   |   ^
